src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -26,31 +27,35 @@ const propTypes = {
     /** Whether to show welcome text on a particular page */
     shouldShowWelcomeText: PropTypes.bool.isRequired,
 
-    /** SafeArea insets */
-    insets: PropTypes.shape(PropTypes.object),
-
     ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
 };
 
-const defaultProps = {
-    insets: {},
-};
-
-const SignInPageLayoutNarrow = props => (
+const SignInPageContent = props => (
     <ScrollView
         keyboardShouldPersistTaps="handled"
         showsVerticalScrollIndicator={false}
         style={[
             styles.h100,
             styles.alignSelfCenter,
+            !props.isSmallScreenWidth && styles.signInPageWideLeftContainer,
+        ]}
+        contentContainerStyle={[
+            scrollViewContentContainerStyles,
+            !props.isSmallScreenWidth && styles.ph6,
         ]}
-        contentContainerStyle={scrollViewContentContainerStyles}
     >
-        <Form style={[styles.flex1, styles.signInPageNarrowContentContainer, styles.alignSelfStretch, styles.ph5]}>
+        <Form style={[
+            styles.flex1,
+            styles.alignSelfStretch,
+            props.isSmallScreenWidth && styles.signInPageNarrowContentContainer,
+            props.isSmallScreenWidth ? styles.ph5 : styles.ph4,
+        ]}
+        >
```

Comment:
Indentation looks a little weird here. Maybe `]}>`?

Reply:
Actually, it's totally fine. There are two ways here. 
This one 
 e.g. https://github.com/Expensify/App/blob/5362f017468a300b263d35bb257102e7d45c9ca8/src/pages/home/report/ReportActionCompose.js#L499-L506
and
```
        <Form
            style={[
                styles.flex1,
                styles.alignSelfStretch,
                props.isSmallScreenWidth && styles.signInPageNarrowContentContainer,
                props.isSmallScreenWidth ? styles.ph5 : styles.ph4,
            ]}
        >
```

Reply:
Huh, Ok well that's fine with me. This was a NAB.